.gitlab-ci.yml

```diff
@@ -194,6 +193,8 @@ test:unit:mac:
 
 publish:tests:
   stage: publish
+  # Keep golang version aligned with latest yocto release
+  image: golang:1.20-alpine3.17
```

Comment:
Q: So not kirkstone? I'm pretty sure that is 1.17

Reply:
Thanks. It is the comment that is off here, I meant to add such comment to `test:smoketests:linux:`. For this job, it is okey to use newer golang (and it is convenient to use newer to get `goveralls` working properly).

See rework.

Reply:
Makes sense now :)